File: src/com/totalplay/ffm/plantainterna/service/ConsultaOTService.java
package com.totalplay.ffm.plantainterna.service;

import com.totalplay.ffm.plantainterna.model.consultaOTPI.ParamConsultaOTPI;
import com.totalplay.ffm.utilerias.model.DataTableResponse;
import com.totalplay.ffm.utilerias.model.ServiceResponseResult;

public interface ConsultaOTService {
	public ServiceResponseResult consultaFiltros(String params);

	public DataTableResponse consultaOT(ParamConsultaOTPI paramsOT);

	public ServiceResponseResult consultaImagenesOt(String params);

	public ServiceResponseResult consultaInformacionDetalleOt(String params);

	public ServiceResponseResult consultaMaterialesOts(ParamConsultaOTPI paramsOT);

	public ServiceResponseResult getComentariosIntegrador(String params);

	public ServiceResponseResult historico(String params);

	public ServiceResponseResult consultaActividadTecnico(String params);

	public ServiceResponseResult consultaInfoTrayectoria(String params);

	public ServiceResponseResult consultaInformacionRed(String params);

	public ServiceResponseResult consultaCambioEquipo(String params);

}

File: webapp/WEB-INF/resources/js/plantainterna/consultaOT/consultaOTService.js
app.service("consultaOTService", function($http){

	this.consultaOt = function (params) {
        return $http({
            method: "post",
            url: "req/consultaOT",
            data: params,
            headers: {'Content-Type': "application/json; charset=utf-8"},
            transformRequest: angular.identity
        });
    };

    this.consultaImagenesOt = function (params) {
        return $http({
            method: "post",
            url: "req/consultaImagenesOt",
            data: params,
            headers: {'Content-Type': "application/json; charset=utf-8"},
            transformRequest: angular.identity
        });
    };

    this.consultaInfoDetalle = function (params) {
        return $http({
            method: "post",
            url: "req/consultaInformacionDetalleOt",
            data: params,
            headers: {'Content-Type': "application/json; charset=utf-8"},
            transformRequest: angular.identity
        });
    };

    this.consultaMaterialOt = function (params) {
        return $http({
            method: "post",
            url: "req/consultaMaterialesOts",
            data: params,
            headers: {'Content-Type': "application/json; charset=utf-8"},
            transformRequest: angular.identity
        });
    };

    this.consultaComentarios = function (params) {
        return $http({
            method: "post",
            url: "req/getComentariosIntegrador",
            data: params,
            headers: {'Content-Type': "application/json; charset=utf-8"},
            transformRequest: angular.identity
        });
    };

    this.consultaHistorico = function (params) {
        return $http({
            method: "post",
            url: "req/historico",
            data: params,
            headers: {'Content-Type': "application/json; charset=utf-8"},
            transformRequest: angular.identity
        });
    };

    this.consultaActividad = function (params) {
        return $http({
            method: "post",
            url: "req/consultaActividadTecnico",
            data: params,
            headers: {'Content-Type': "application/json; charset=utf-8"},
            transformRequest: angular.identity
        });
    };

    this.consultaTrayectoria = function (params) {
        return $http({
            method: "post",
            url: "req/consultaInfoTrayectoria",
            data: params,
            headers: {'Content-Type': "application/json; charset=utf-8"},
            transformRequest: angular.identity
        });
    };

    this.consultaInfoRed = function (params) {
        return $http({
            method: "post",
            url: "req/consultaInformacionRed",
            data: params,
            headers: {'Content-Type': "application/json; charset=utf-8"},
            transformRequest: angular.identity
        });
    };

    this.consultarcambioDeEquipo = function (params) {
        return $http({
            method: "post",
            url: "req/consultaCambioEquipo",
            data: params,
            headers: {'Content-Type': "application/json; charset=utf-8"},
            transformRequest: angular.identity
        });
    };
})